Sources/TuistKit/Models/Product.swift

```diff
@@ -6,6 +6,7 @@ enum Product: String, CustomStringConvertible, CaseIterable {
     case staticLibrary = "static_library"
     case dynamicLibrary = "dynamic_library"
     case framework
+    case staticFramework
```

Comment:
This is a little bit of a lie, since technically there's no staticFramework product - should organise this differently?

Reply:
This is an interesting one, even though it doesn't map to a real product, tuist acts a helper tool, as such it can have this defined here for convenience. 

Luckily tuist generates Xcode projects and doesn't yet have a need to read them like the underlying xcodeproj library.

Reply:
What about just using the existing `static_library`? That's what it is in the generated project. The only difference is that we add support for resources, which Xcode projects doesn't support for libraries. I don't know how confusing it'd be for users to see a library that supports resources.

Reply:
Static frameworks are very handy for mixed language static frameworks, very convenient for managing headers even if there are no resources involved. We often find ourselves applying this method of having a framework product with a modified `MACH_O_TYPE` build setting, having a convenience for it is worth consideration.